File: 2017-multi-jpa-java/app/src/main/java/sample/member/app/web/signup/SignupViewController.java
package sample.member.app.web.signup;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

@Controller
public class SignupViewController {

	@RequestMapping(value = "/signup", method = RequestMethod.GET)
	public String signupView() {
		return "signup";
	}

}

File: member_crud-jpa-kotlin/app/src/main/kotlin/sample/member/app/web/signup/SignupViewController.kt
package sample.member.app.web.signup

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RequestMethod

@Controller
class SignupViewController {

	@RequestMapping(value = ["/signup"], method = [RequestMethod.GET])
	fun signupView(): String = "signup.html"

}
